<?xml version="1.0"?>
<launch>
<param name="use_sim_time" value="true" />
<!-- <node pkg="joy" type="joy_node" name="joy_node" /> -->
<node pkg="mono_control" type="control_node.py" name="control_node" output="screen">
    <param name="enable_back" value="false"/>
    <param name="max_vx" value="0.1"/>
    <param name="max_va" value="0.1"/>
    <param name="gain_vx" value="0.1"/>
    <param name="gain_va" value="0.1"/>
    <param name="distance" value="1.0"/>
    <param name="timeout" value="0.5"/>
  </node>
</launch>